packages/framework/azure-service-utils/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { generateToken } from "@fluidframework/server-services-client";
```

Comment:
Devs also need ScopeType for defining permissions.

```suggestion
export { ScopeType } from "@fluidframework/protocol-definitions";
export { generateToken } from "@fluidframework/server-services-client";
```

Reply:
Added this + dependency update